libs/server/Resp/Bitmap/BitmapCommands.cs

```diff
@@ -108,7 +108,7 @@ internal sealed unsafe partial class RespServerSession : ServerSessionBase
         /// The bit is either set or cleared depending on value, which can be either 0 or 1.
         /// When key does not exist, a new key is created.The key is grown to make sure it can hold a bit at offset.
         /// </summary>
-        private bool StringSetBit<TGarnetApi>(byte* ptr, ref TGarnetApi storageApi)
+        private bool StringSetBit<TGarnetApi>(int count, byte* ptr, ref TGarnetApi storageApi)
```

Comment:
I don't think we need to introduce a count here. Ideally, the ACL check should happen at the parsing level or right after. See  RespCommand.cs